### PR TITLE
Comment debug

### PR DESCRIPTION
**Pull request by max-friedman**
- User nicknames now show instead of userID on comments

### CODE REVIEW

File: project/src/main/java/com/google/sps/Servlets/CommentServlet.java [hunk 1]
```diff
@@ -52,3 +57,15 @@ private String convertToJson(Object obj) {
     return json;
   }
 }
+
+class CommentWithName {
```
Comment (g-luo) [inline project/src/main/java/com/google/sps/Servlets/CommentServlet.java:61]: why can't you just modify the comment class to have a name field that has an associated getter / setter?

**Review by tin0d0 — APPROVED** [review]
lgtm!

File: project/src/main/java/com/google/sps/Servlets/CommentServlet.java (hunk 1 above)
Comment (g-luo) [reply project/src/main/java/com/google/sps/Servlets/CommentServlet.java:61]: then in your for loop when you're associating the name you don't need a new class

Comment (max-friedman) [reply project/src/main/java/com/google/sps/Servlets/CommentServlet.java:61]: That would also work but I would have to go back and refactor several functions in the database, I think our time is better spend fixing other bugs and nitpicks right now

**Pull request merged**